src/server.jl

```diff
@@ -892,7 +892,7 @@ function close!(server::Server, path::String)
     borrow_file!(server, path) do file
         Malt.stop(file.worker)
         lock(server.lock) do
-            delete!(server.workers, path)
+            pop!(server.workers, file.path)
```

Comment:
What's the reason for this change?

Reply:
`pop!` errors if the key is not found, `delete!` doesn't. That hid a bug because `path` isn't necessarily the same as the one used to key the dict.